README.md

```diff
@@ -31,6 +31,8 @@ which is used to store the test logs under `tests/conformance/data`.
 
 A CLI tool for interacting with the format is available [here](./go/cli/mcap).
 
+Download the [latest release](https://github.com/foxglove/mcap/releases/latest).
```

Comment:
I wonder if maybe we should just change the link above this one to point here, and remove the new line. Probably people who are following the "is available" link are looking for a download, not to browse code.